backend/tasks/views.py

```diff
@@ -396,6 +410,82 @@ def delete_project_tasks(self, request, pk=None):
                 }
             )
 
+    @swagger_auto_schema(
+        method="post",
+        request_body=openapi.Schema(
+            type=openapi.TYPE_OBJECT,
+            properties={
+                "user_id": openapi.Schema(type=openapi.TYPE_INTEGER),
+                "task_type": openapi.Schema(type=openapi.TYPE_STRING),
+            },
+            required=["user_id"],
+        ),
+        manual_parameters=[
+            openapi.Parameter(
+                "page",
+                openapi.IN_QUERY,
+                description=("A integer refering to page no of paginated response"),
+                type=openapi.TYPE_INTEGER,
+                required=False,
+            ),
+            openapi.Parameter(
+                "records",
+                openapi.IN_QUERY,
+                description=(
+                    "A integer refering to no of records in single page of a paginated response"
+                ),
+                type=openapi.TYPE_INTEGER,
+                required=False,
+            ),
+        ],
+        responses={
+            200: "Returns a paginated list of recent tasks annotated/reviewed by a user",
+            403: "Not authorized!",
+            400: "Invalid parameters in the request body!",
+        },
+    )
+    @action(
+        detail=False,
+        methods=["POST"],
+        url_path="annotated_and_reviewed_tasks/get_users_recent_tasks",
+        url_name="get_users_recent_tasks",
+    )
+    def get_users_recent_tasks(self, request):
+        try:
+            user_id = request.data.get("user_id")
+            task_type = request.data.get("task_type", "annotation")
+
+            user = User.objects.get(pk=user_id)
+
+            annotations = Annotation.objects.filter(completed_by=user)
+            if task_type == "review":
+                annotations = annotations.filter(parent_annotation__isnull=False)
+            else:
+                annotations = annotations.filter(parent_annotation__isnull=True)
+
+            annotations = annotations.order_by("-updated_at")
+            annotations = self.paginate_queryset(annotations)
+
+            response = []
+
+            for annotation in annotations:
+                data = {
+                    "Project ID": annotation.task.project_id.id,
+                    "Task ID": annotation.task.id,
+                    "Updated at": utc_to_ist(annotation.updated_at),
```

Comment:
![image](https://user-images.githubusercontent.com/26191118/194214327-ca977f40-0081-47b9-a162-59463d67f849.png)

We should return the updated_at field in a user-readable form.  (Example 02/10/2022 15:43 [IST])